### PR TITLE
Add Rubik One

### LINKED ISSUE #66: Font Request: Rubik One
### Font Name

Rubik One

### Link to font source

https://github.com/google/fonts/tree/main/ofl/rubikone

### Link to font license

https://github.com/google/fonts/blob/main/ofl/rubikone/OFL.txt

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by dgadelha**
Font was downloaded from Google Fonts repository (https://github.com/google/fonts/tree/main/ofl/rubikone) and converted using [this tool](https://www.fontsquirrel.com/tools/webfont-generator) with **Basic** method (`Straight conversion with minimal processing.`)

Closes #66

### CODE REVIEW

**Comment by ayuhito** [comment]
I'm trying to understand why Rubik One was delisted from Google Fonts but there doesn't seem any public record. I can see it varies from Rubik Mono One since that doesn't support lowercase characters. I think it's fine we include this.

File: fonts/other/rubik-one/scss/mixins.scss [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    $axes,
+    if($axes == all, full, $axes),
+    if($isVariable, if(map.has-key($metadata, axes, wght), wght, full), null)
+  );
+
+  @each $subset in $subsets {
+    @each $unicodeSubset, $unicodeRange in map.get($metadata, unicode) {
+      @if (
+        ($subset == $unicodeSubset) or
+          (
```
Comment (ayuhito) [inline fonts/other/rubik-one/scss/mixins.scss:77]: Can you regenerate this package with the latest commit in this repo or the latest version of `@fontsource-utils/cli`. We pushed a bugfix for this file.

**Review by ayuhito — CHANGES_REQUESTED** [review]

File: fonts/other/rubik-one/scss/mixins.scss (hunk 1 above)
Comment (dgadelha) [reply fonts/other/rubik-one/scss/mixins.scss]: Updated :)

**Review by ayuhito — APPROVED** [review]

**Comment by ayuhito** [comment]
Thank you! 🎉

**Pull request merged**